src/Couchbase.Lite.Tests.Shared/URLEndpointListenerTest.cs

```diff
@@ -141,7 +141,7 @@ public void TestBusyPort()
             _listener = CreateListener(false);
             //listener1 uses the same port as listener
             var config = CreateListenerConfig(false);
-            var listener1 = Listen(config, PosixBase.GetCode(nameof(PosixWindows.EADDRINUSE)), CouchbaseLiteErrorType.POSIX);
+            var listener1 = Listen(config, (int)C4NetworkErrorCode.AddressNotAvailableAIL, CouchbaseLiteErrorType.POSIX);
```

Comment:
Wrong domain?

Reply:
Currently there are 4 domains to choose from: CouchbaseLite, POSIX, SQLite, and Fleece. Are we adding new type such as Network?

Reply:
If this is on the public side of things, then currently network domain are translated to the CouchbaseLite domain with an error code offset.